src/pages/Main/BlackWindow/BlackWindow.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './BlackWindow.scss';
+
+function BlackWindow({ guideModalClose }) {
+  return (
+    <div
+      className="blackWindow"
+      onClick={() => {
+        guideModalClose();
+      }}
+    />
+  );
```

Comment:
```suggestion
    <div
      className="blackWindow"
      onClick={guideModalClose}
    />
  );
```
다른 파일들에서도 이렇게 사용하세요!